cohorts/load.py

```diff
@@ -215,10 +223,16 @@ def __init__(self,
                  join_how="inner",
                  check_provenance=False,
                  polyphen_dump_path=None,
-                 pageant_coverage_path=None):
+                 pageant_coverage_path=None,
+                 variant_type="snv",
```

Comment:
Yeah, it became cumbersome and wordy to pass it around everywhere. Agree it's a little weird--